### DISCUSSION TITLE
How to connect the PF scatters in mtex?

### DISCUSSION THREAD

**Discussion by SingPr** [mtex-toolbox/mtex#1547] [Ask Anything]
Dear community,
    Now I have a PF scatter like below.
![untitled](https://user-images.githubusercontent.com/89964907/189601878-6d312b3b-c976-4fd2-8062-f3d4b6e5f58f.jpg)
I want to connect three scattered points with a trace line on the surface of the sphere.  It's definitely not a straight line in this case. So, what should I do in MTEX?
thanks
Sing

**Comment by kilir** [comment]
Hi,
how about:
```
v = vector3d.rand(3)
plot(v)
hold on
line(v(1),v(2),'add2all')
line(v(2),v(3),'add2all')
line(v(3),v(1),'add2all')
hold off
```
Cheers,
Rüdiger

**Reply by SingPr** [reply]
Hi Rüdiger,
Thank you for your quick reply. But I don't know how to add code. My codes are as follows:

[grains,ebsd.grainId] = calcGrains(ebsd('indexed'),'threshold',[2degree, 10degree]);
grains = smooth(grains,1);
gB = grains.boundary;
% plot the pole figure of the selected grain, ( scatters )
plotPDF(grains(13).meanOrientation, Miller(1,0,0, grains.CS),'Marksize',2,'MarkerColor','red')
hold on

The running result is shown above. So I wanna connect three scattered points(red).

Thanks
Sing

**Reply by kilir** [reply]
You could try to adapt the following
```
h = symmetrise(Miller(1,0,0,cs),'antipodal','unique')
v = grains(13).meanOrientation* h

plot(v)
hold on
line(v(1),v(2),'add2all')
line(v(2),v(3),'add2all')
line(v(3),v(1),'add2all')
hold off
```
Cheers,
Rüdiger

**Reply by SingPr** [reply]
Dear Rüdiger,
    It works well, thanks a lot. 
Have a good day.
Sing
